doc/man3/X509_STORE_CTX_new.pod

```diff
@@ -103,8 +112,9 @@ to the verification parameters associated with I<ctx>.
 X509_STORE_CTX_get0_untrusted() retrieves an internal pointer to the
 stack of untrusted certificates associated with I<ctx>.
 
-X509_STORE_CTX_set0_untrusted() sets the internal point to the stack
+X509_STORE_CTX_set0_untrusted() sets the internal pointer to the stack
 of untrusted certificates associated with I<ctx> to I<sk>.
+X509_STORE_CTX_verify() will take the first element, if any, as its default target.
```

Comment:
Please use `, as its target if the target certificate is not set explicitly.`

To make it clear.